pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -2727,6 +2727,51 @@ func TestReconcileWorkspaceMissing(t *testing.T) {
 	}
 }
 
+// TestReconcileDefaultWorkspace tests a reconcile of a TaskRun that does
+// not include a Workspace that the Task is expecting.
+func TestReconcileDefaultWorkspace(t *testing.T) {
```

Comment:
@sbwsg included this test for when default is added, and right above is a test for when workspace is missing and the default is not provided